lib/sinon/util/core/called-in-order.js

```diff
@@ -1,15 +1,34 @@
 "use strict";
 
+var every = Array.prototype.every;
+
 module.exports = function calledInOrder(spies) {
+    var callMap = {};
+
+    function hasCallsLeft(spy) {
+        if (callMap[spy.id] === undefined) {
+            callMap[spy.id] = 0;
+        }
+
+        return callMap[spy.id] < spy.callCount;
+    }
+
     if (arguments.length > 1) {
-        spies = arguments;
+        spies = Array.prototype.slice.call(arguments);
```

Comment:
@fearphage well noticed! I just fixed it.

Thanks for your careful review, this was indeed something that should be fixed 👍